app/controllers/stripe/events_controller.rb

```diff
@@ -5,7 +5,7 @@ class EventsController < ::Stripe::ApplicationController
 
     def create
       @event = dispatch_stripe_event params
-      respond_with @event, :location => nil
+      render json: @event, location: nil
```

Comment:
Ok great will do that! I did test with `head :ok` earlier, and it worked fine. I did notice that Stripe will save the results of the webhook response and you can see it in their web UI in case something goes wrong. So maybe that could be useful, but doesn't look like `stripe-rails` was using that

Reply:
Ok I updated the PR with `head :ok`